.github/workflows/reusable-test.yaml

```diff
@@ -113,3 +113,16 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           path: ./logs
+
+  codespell:
+    name: Check spelling
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup environment
+        uses: ./.github/workflows/actions/setup-environment
```

Comment:
[The workflow has unfortunately failed](https://github.com/LayerZero-Labs/devtools/actions/runs/8430221261/job/23116118945?pr=479) - the problem is that this job is missing the first step:

```yaml
steps:
  - name: Checkout repo
     uses: actions/checkout@v4
```

Once you put that one in you should be good and green!